articles/install-update-qdk.md

```diff
@@ -154,6 +154,7 @@ Select your development environment below.
 * [Visual Studio: update the QDK extension](#update-the-visual-studio-qdk-extension)
 * [VS Code: update the QDK extension](#update-the-vs-code-qdk-extension)
 * [Command line and C#: update project templates](#c-using-the-dotnet-command-line-tool)
+* [Python: Update the Python SDK for Optimization](#update-the-python-sdk-for-optimization)
```

Comment:
Is the official name of the SDK the 'Python SDK for Optimization'? Because the [published package name](https://pypi.org/project/azure-quantum/) seems to be the 'Python SDK for Azure Quantum'.

Reply:
Good point. We say "Python SDK for Azure Quantum" elsewhere, so let's use that here and then we can always amend it if we get feedback to that says this is confusing. https://github.com/MicrosoftDocs/quantum-docs/blob/main/articles/optimization-install-sdk.md